zio-cli/shared/src/main/scala/zio/cli/BuiltIn.scala

```diff
@@ -1,7 +1,14 @@
 package zio.cli
 
+import zio.IO
+
 final case class BuiltIn(help: Boolean, shellCompletions: Option[ShellType])
 object BuiltIn {
-  val options: Options[BuiltIn] =
-    (Options.bool("help", true) :: ShellType.option.optional("N/A")).as(BuiltIn(_, _))
+  trait BuiltInOptions {
```

Comment:
I prefer to avoid inheritance when possible. Can you define these directly in `Command`?